backend/app/views/secondary_donations/_secondary_donation.json.jbuilder

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
 json.id secondary_donation.id
-json.couponId secondary_donation.coupon_id
 json.amount secondary_donation.amount
 json.campaignCharityId secondary_donation.campaign_charity_id
+
+if secondary_donation.coupon
+  json.urlToken secondary_donation.coupon.url_token
+else
+  json.urlToken nil
+end
```

Comment:
why this change? does it affect anything?

Reply:
Checked the frontend and nowhere do we use couponId. Created #173